serve/benchmarks/benchmark_throughput.py

```diff
@@ -179,7 +178,6 @@ def main(args: argparse.Namespace):
     parser.add_argument("--local-id", type=str, required=True)
     parser.add_argument("--artifact-path", type=str, default="dist")
     parser.add_argument("--use-staging-engine", action="store_true")
-    parser.add_argument("--max-num-batched-tokens", type=int, default=-1)
```

Comment:
Need to add `max_num_sequences` if you remove this

Reply:
yep just figured it out so added :)